server/src/auth/JWT.cc

```diff
@@ -22,8 +22,8 @@ namespace server::auth {
                 .set_not_before(now)
                 .set_expires_at(now + std::chrono::days{30}) //TODO: move to config
                 .set_type("JWS")
-                .set_payload_claim("userId", boost::json::value(payload->userId))
-                .set_payload_claim("role", boost::json::value(payload->role))
+                .set_payload_claim("userId", Traits::value_type{*payload->userId})
+                .set_payload_claim("role", Traits::value_type{*payload->role})
```

Comment:
You shouldn't do that
use equivalent of `nlohmann_json` - sth like `nlohmann_json::json::value`

Reply:
That is what `Traits::value_type` aliases to.

Reply:
Okay? But the alias is to the `jwt::traits::nlohmann_json::value_type` not to `nlohmann::json::value`...

Reply:
> Okay? But the alias is to the `jwt::traits::nlohmann_json::value_type` not to `nlohmann::json::value`...

And what is the issue with that?

Reply:
nvm, they used `.set_payload_claim("sample", jwt::claim(std::string("test")))`
we should also use `jwt::claim()` like this:
```c++
set_payload_claim("userId", jwt::claim{*payload->userId})
set_payload_claim("role", jwt::claim{*payload->role})
```

Reply:
Okay, will you change this in the next commit?

Reply:
yes